setup.py

```diff
@@ -1,8 +1,16 @@
 from os.path import realpath, dirname, join
```

Comment:
The code below currently wouldn't work, because you are calling  `os.path.join` instead of `join` and so forth. Can you just change this line to

    import os

Reply:
Done